base/include/AudioToTextXForm.h

```diff
@@ -0,0 +1,47 @@
+#pragma once
+
+#include "Module.h"
+
+// size of audio to process should be a parameter. 
+// Cache variable to collect frames for processing
+
+class AudioToTextXFormProps : public ModuleProps
+{
+public:
+	enum DecoderSamplingStrategy {
+		GREEDY,  
+		BEAM_SEARCH
+	};
+	AudioToTextXFormProps(
+		DecoderSamplingStrategy _samplingStrategy,
+		std::string _modelPath,
+		int _bufferSize);
+	DecoderSamplingStrategy samplingStrategy;
+	std::string modelPath;
+	int bufferSize;
```

Comment:
The dynamic property change of the module requires serializing the props class. So, boost serialization needs to be added here for it to work. For reference see `Mp4WriterSink.h` file. 

It will be best to have another test for dynamic prop change as well assuming we want to support that in the module.

Reply:
Since, handlePropChange is implemented in the module class, I am assuming we are planning to support dynamic change of props

Reply:
Done, but not sure if the test is running. Can you please review?

Also the only parameter change that makes sense is buffer size

Reply:
done. Although changing modelPath during run time does not make sense. So have not tested that.

Reply:
What you can do is - throw an error if that prop changes with a error saying dynamic change of modelpath is not supported

Reply:
Done. Added UT also